test/k8sT/Nightly.go

```diff
@@ -442,11 +439,29 @@ var _ = Describe("NightlyExamples", func() {
 		})
 
 		for _, image := range helpers.NightlyStableUpgradesFrom {
-			func(image string) {
-				It(fmt.Sprintf("Update Cilium from %s to master", image), func() {
-					By("Cleaning up everything before testing an upgrade from %s", image)
+			func(version string) {
+				It(fmt.Sprintf("Update Cilium from %s to master", version), func() {
+					By("Cleaning up everything before testing an upgrade from %q", version)
 					cleanupPreTest(image)
 
+					By("Validating that cilium can be installed")
+					canRun, err := helpers.CanRunK8sVersion(version, helpers.GetCurrentK8SEnv())
+
```

Comment:
Maybe we can move this to `ValidateCiliumUpgrades` to avoid duplicated code.

Reply:
Is in there, but Validating cilium does not install cilium on that version, so that check need to happens in there.

Reply:
I mean from line 447 to line 463

Reply:
do the installation in the ValidateCiliumUpgrades?

Reply:
@aanm done